ci-requirements.txt

```diff
@@ -1,12 +1,13 @@
-setuptools
-certifi
-requests
-cryptography
-typing_extensions
-aiohttp
-aiodns
-yarl
-ruff
-tox
+setuptools>=60.9.0
+certifi>=2018.1.18
+requests>=2.18.4
+cryptography>=2.6.1
+typing_extensions>=4.4.0
+aiohttp<=3.10.11
```

Comment:
maybe we only set version for aiohttp but not for others?

Reply:
@ttodua it's not an issue to have the versions here and most of them are loosely referenced